test/non-functional/auth/my-privileges/common.ts

```diff
@@ -88,6 +107,8 @@ const read_appl_join_Privilege = [...readPrivilege, ...apply_join_Privilege];
 
 export const crud_sortPrivileges = crudPrivilege.sort();
 export const sortPrivileges = cgrudPrivilege.sort();
+export const sort_createDiscussion_Privileges = cgrudPrivilege.sort();
```

Comment:
Variables shouldn't be named in imperative mood. `sort_...` is a name for a function that sorts something. Change it to `sorted_createDiscussion_Privileges`.